docs/interfaces/desktopagent.md

```diff
@@ -134,12 +198,38 @@ ___
 
 ▸ **resolve**(intent: *`String`*, context?: *[Context](../#context)*): `Promise`<`Array`<[ActionMetadata](actionmetadata.md)>>
 
-*Defined in [interface.ts:93](/src/interface.ts#L93)*
+*Defined in [interface.ts:140](/src/interface.ts#L140)*
 
 Resolves an intent & context pair to a mapping of Intents and Apps (action metadata).
 
 Resolve is effectively granting programmatic access to the Desktop Agent's resolver. Returns a promise that resolves to an Array. The resolved dataset & metadata is Desktop Agent-specific. If intent argument is falsey, then all possible intents - and apps corresponding to the intents - are resolved for the provided context. If the resolution errors, it returns an `Error` with a string from the `ResolveError` enumeration.
 
+```javascript
+//find what Apps can support SartChat with the given context structure
+var intentR = await agent.resolve("StartChat",{version:"1.0.0",entities:[
+   {
+      "type":"Contact",
+      "name":"Nick",
+      "id":{
+           "default":"nick@openfin.co",
+           "email":"nick@openfin.co"
+      }
+   }
+]});
+
+//find what Intents and Apps are supported for a given context
+var actionR = await agent.resolve(false,{version:"1.0.0",entities:[
```

Comment:
Why `false`? Do you mean `null`? I.e. what is the convention for not specifying the intent name - I think we need to be very clear about this use case.